src/process/filter/index.ts

```diff
@@ -12,18 +12,31 @@ export const filterProcessSync: ProcessorFnSync<FilterScope> = (context: FilterC
         const modelType = Utils.getModelType(component);
         switch (modelType) {
             case 'dataObject':
-                scope.filter[absolutePath] = {data: {}};
+                scope.filter[absolutePath] = {
+                    compModelType: modelType,
+                    include: true,
```

Comment:
"include" is really not needed. By this having a "scope.filter" for that path, it is included.  Are you maybe adding the ability for some other processor to NOT include it?